code/src/App.js

```diff
@@ -1,9 +1,74 @@
-import React from 'react'
+import React, { useEffect, useState } from 'react'
+
+import { Card } from 'components/Card'
+import { MessageInput } from 'components/MessageInput'
 
 export const App = () => {
+  // App is responsible for fetching, listing, and posting the messages – as well as contains the like function (it's here because )
+  const MESSAGES_URL = "https://happy-thoughts-technigo.herokuapp.com/thoughts"
+  const [messages, setMessages] = useState([])
+
+  useEffect(() => {
+    fetchMessages()
+  }, [])
+
+  const fetchMessages = () => {
+    fetch(MESSAGES_URL)
+      .then((res) => {
+        // This is the response we're getting – but we need to transform it to JSON.
```

Comment:
Love all the comments. Makes it really easy to follow your code and understand whats going on.